### ISSUE TITLE
Projects with custom blocks don't load

### ISSUE THREAD

**Issue by ivanov** [NoMod-Programming/Whiskers#4]
First, thank you for taking the time to make Whiskers! 

This might be an issue with how they are saved, but custom blocks cause an error on project load.

In ScriptableScratchMorph CustomBlockFromTuple - on this line
```
    ifFalse: [t3 := {t1 second. t1 first = #callReporter
```

 the variables are
```
t1: 	#('call')
t2: 	a ScratchSpriteMorph(3251934)
t3-t9: nil
```
So the `t1 second.` portion causes a "Subscript is out of bounds: 2" error

The way to reproduce this is to make a custom block *AND* use at least one of the custom blocks in the sprite code. (having a custom block defined but not used seems to not cause the error)

**Comment by NoMod-Programming** [comment]
Well, this is embarassing. I misspelled "asBlockTuple" in the source code. I'll go ahead and release a version that fixes this, but looking at the source, because of that misspelling, custom blocks didn't save correctly. If you want, I can make Whiskers load the file, but the custom blocks will need to be re-created (Just the blocks that call them). Let me know if you want this.

**Comment by ivanov** [comment]
Oh, glad you found the bug! I don't need a workaround for files saved with version of Whiskers that had the bug, so no need to bother with that if that makes cutting a new release easier for you.

**Comment by NoMod-Programming** [comment]
Fixed in 0.9a6

**Comment by benduckwdesign** [comment]
Hi, I just wanted to let you know that the website download for standalone Whiskers is 0.9a5 and needs to be updated on your website, I ran into this issue early and it was pretty frustrating to realize I had an outdated version. I started this project in Whiskers so Scratch 2.0 wasn't involved.

<img width="1675" height="866" alt="Image" src="https://github.com/user-attachments/assets/a843274d-4777-4f85-bdd8-9c4469258f3b" />
